server/watsonAPI/watsonResults/transcribeTest

```diff
@@ -0,0 +1,91 @@
+But. 
```

Comment:
Not sure if this needs to be included into the branch. Only because we've got example tone data in the static directory of this branch